department-of-veterans-affairs/vets-website
# Redirect when travel pay feature flag is disabled

## Summary

Implements the `travel_pay_power_switch` feature flag in the frontend. Redirects back to the homepage if the feature flag is not enabled.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/78307

## Testing done

- Ensure that `travel_pay_power_switch` feature flag is disabled
- Navigate to `/health-care/file-travel-claim/`
- Observe that site redirects to homepage

- Enable `travel_pay_power_switch` feature flag
- Navigate to `/health-care/file-travel-claim/`
- Page loads as expected
